webapp/src/containers/custom_post/sericeNowPost.tsx

```diff
@@ -72,13 +80,23 @@ const NotificationPost = ({post}: NotificationPostProps) => {
                     key={field.title}
                     className='shared-post__field-value'
                 >
-                    <span>
-                        {field.value}
-                    </span>
+                    {field.title === 'Description' ? (
+                        <span>
+                            {description}
+                        </span>
+                    ) : (
+                        <span>
+                            {(type as string) === 'custom_sn_share' ? Utils.getRecordValueForHeader(field.title as TypesContainingLink, field.value) : (field.value)}
```

Comment:
What's the need for checking this condition here? `(type as string) === 'custom_sn_share'`

Reply:
we are showing navigable links of some fields when a record is shared and field value when for notification post